openapi_python_client/parser/properties/__init__.py

```diff
@@ -439,8 +444,12 @@ def _property_from_data(
     if isinstance(data, oai.Reference):
         return _property_from_ref(name=name, required=required, parent=None, data=data, schemas=schemas)
 
+    sub_data: List[Union[oai.Schema, oai.Reference]] = []
+    for _data in [data.allOf, data.anyOf, data.oneOf]:
+        if isinstance(_data, Iterable):
```

Comment:
```suggestion
    for _data in (data.allOf, data.anyOf, data.oneOf):
        if data:
```
Should be a bit quicker, although same question about defaulting to empty list.

Reply:
Resolved in https://github.com/triaxtec/openapi-python-client/pull/426/commits/63d0bd7891759300592a20592991f4ec842f0a84 , same answer, locally we are good, it was already defaulting to an empty list:

https://github.com/triaxtec/openapi-python-client/blob/0f58cfd4fc985ba871b3d2694d861c7f595dcb81/openapi_python_client/parser/properties/__init__.py#L443

As did not seen explicit `None` check on `anyOf` nor `oneOf` else where